src/modules/landing_target_estimator/LandingTargetEstimator.cpp

```diff
@@ -295,14 +335,26 @@ void LandingTargetEstimator::_update_params()
 	param_get(_paramHandle.scale_x, &_params.scale_x);
 	param_get(_paramHandle.scale_y, &_params.scale_y);
 
-	int32_t sensor_yaw = 0;
-	param_get(_paramHandle.sensor_yaw, &sensor_yaw);
-	_params.sensor_yaw = static_cast<enum Rotation>(sensor_yaw);
+	int32_t sensor_rot = 0;
```

Comment:
Why the rename from `yaw` to `rot`?

Reply:
The parameter is called  LTEST_SENS_ROT.
At first I wanted this parameter to also be used with the UWB sensor.
But after speaking with Daniel we decided that this should be published in the sensor_uwb uorb message
I can change it back.